conductr_cli/conductr_backup.py

```diff
@@ -0,0 +1,232 @@
+import hashlib
+import json
+import logging
+import os
+import shutil
+import time
+from shutil import rmtree
+
+import requests
+from requests_toolbelt import MultipartDecoder
+
+from conductr_cli import conduct_url, conduct_request, validation
+from conductr_cli.bndl_utils import file_write_bytes, file_write_string
+from conductr_cli.bundle_core_info import BundleCoreInfo
+from conductr_cli.conduct_info_inspect import filter_bundles_by_id_or_name
+from conductr_cli.conduct_url import conductr_host
+from conductr_cli.exceptions import ConductBackUpError
+from conductr_cli.http import DEFAULT_HTTP_TIMEOUT
+
+
+@validation.handle_connection_error
+@validation.handle_http_error
+@validation.handle_wait_timeout_error
```

Comment:
Have we got a validation handler for `ConductBackupError`? I can't seem to find it in this PR.

Once we have this, we can annotate `@validation.handle_backup_error` and the method `handle_backup_error` can handle the `ConductBackupError`, display the error message nicely, and ensures the return code is non-zero when `ConductBackupError` is raised.

Here's an example of the error handler which handles `ConductrStartupError`:

https://github.com/typesafehub/conductr-cli/blob/master/conductr_cli/validation.py#L81